ui/src/App.tsx

```diff
@@ -106,12 +94,9 @@ export default function App() {
   };
   useEffect(() => {
     graphAppSetup();
-    // fetch("/workspace/update_version", {
-    //   method: "POST",
-    // });
     setInterval(() => {
       if (curFlowID.current != null) {
-        const graphJson = localStorage.getItem("workflow");
+        const graphJson = JSON.stringify(app.graph.serialize());
```

Comment:
this is an important chage, it ensures accessing the latest graph data, localStorage.getItem("workflow") has latency that it creates some unintentional saveDB updates when switching between workflows